src/web/mod.rs

```diff
@@ -649,6 +680,21 @@ impl_webpage! {
     status = |err| err.status,
 }
 
+#[derive(Debug, Clone, PartialEq, Serialize)]
+pub(crate) struct AxumErrorPage {
+    /// The title of the page
+    pub title: &'static str,
+    /// The error message, displayed as a description
+    pub message: Option<Cow<'static, str>>,
```

Comment:
why an `Option`? seems unfortunate to serve an error page without a message

Reply:
this is coming from the original `ErrorPage` definition, since this is mostly a copy. 

But the point is valid, I changed it